vme/src/unit_data.cpp

```diff
@@ -341,480 +346,493 @@ int unit_data::destruct_classindex()
 {
-    return &open_diff;
+    return &m_open_diff;
 }
 
 void unit_data::setOpenDifficulty(ubit8 value)
 {
-    open_diff = value;
+    m_open_diff = value;
 }
 
 sbit16 unit_data::getNumberOfActiveLightSources() const
 {
-    return light;
+    return m_light;
 }
 
 void unit_data::changeNumberOfActiveLightSourcesBy(sbit16 value)
 {
-    light += value;
+    m_light += value;
 }
 
 void unit_data::setNumberOfActiveLightSources(sbit16 value)
 {
-    light = value;
+    m_light = value;
 }
 
 sbit16 unit_data::getLightOutput() const
 {
-    return bright;
+    return m_bright;
 }
 
 void unit_data::setLightOutput(sbit16 value)
 {
-    bright = value;
+    m_bright = value;
 }
 
 void unit_data::changeLightOutputBy(sbit16 value)
 {
-    bright += value;
+    m_bright += value;
 }
 
 sbit16 unit_data::getTransparentLightOutput() const
 {
-    return illum;
+    return m_illum;
 }
 
 void unit_data::setTransparentLightOutput(sbit16 value)
 {
-    illum = value;
+    m_illum = value;
 }
 
 void unit_data::changeTransparentLightOutputBy(sbit16 value)
 {
-    illum += value;
+    m_illum += value;
 }
 
 ubit8 unit_data::getNumberOfCharactersInsideUnit() const
 {
-    return chars;
+    return m_chars;
 }
 
 void unit_data::decrementNumberOfCharactersInsideUnit()
 {
-    --chars;
+    --m_chars;
 }
 
 void unit_data::incrementNumberOfCharactersInsideUnit()
 {
-    ++chars;
+    ++m_chars;
 }
 
 void unit_data::setNumberOfCharactersInsideUnit(ubit8 value)
 {
-    chars = value;
+    m_chars = value;
 }
 
 ubit8 unit_data::getLevelOfWizardInvisibility() const
 {
-    return minv;
+    return m_minv;
 }
 
 ubit8 *unit_data::getLevelOfWizardInvisibilityPtr()
 {
-    return &minv;
+    return &m_minv;
 }
 
 void unit_data::setLevelOfWizardInvisibility(ubit8 value)
 {
-    minv = value;
+    m_minv = value;
 }
 
 sbit32 unit_data::getMaximumHitpoints() const
 {
-    return max_hp;
+    return m_max_hp;
 }
 
 sbit32 *unit_data::getMaximumHitpointsPtr()
 {
-    return &max_hp;
+    return &m_max_hp;
 }
 
 void unit_data::setMaximumHitpoints(sbit32 value)
 {
-    max_hp = value;
+    m_max_hp = value;
 }
 
 sbit32 unit_data::getCurrentHitpoints() const
 {
-    return hp;
+    return m_hp;
 }
 
 sbit32 *unit_data::getCurrentHitpointsPtr()
 {
-    return &hp;
+    return &m_hp;
 }
 
 void unit_data::setCurrentHitpoints(sbit32 value)
 {
-    hp = value;
+    m_hp = value;
 }
 
 void unit_data::changeCurrentHitpointsBy(sbit32 value)
 {
-    hp += value;
+    m_hp += value;
 }
 
 sbit16 unit_data::getAlignment() const
 {
-    return alignment;
+    return m_alignment;
 }
 
 sbit16 *unit_data::getAlignmentPtr()
 {
-    return &alignment;
+    return &m_alignment;
 }
 
-void unit_data::increaseAlignmentBy(sbit16 value)
+bool unit_data::changeAlignmentBy(int64_t value)
 {
-    alignment += value;
+    return setAlignment(m_alignment + value);
 }
 
-void unit_data::decreaseAlignmentBy(sbit16 value)
+bool unit_data::setAlignment(int64_t value, bool is_vmc)
```

Comment:
It's OK, but I dont think it is necessary to add the isVmc bool.
Kind of think it's an advantage that you can't set the value out of range from anywhere, including vmc.